src/components/views/dialogs/InviteDialog.js

```diff
@@ -432,10 +432,10 @@ export default class InviteDialog extends React.PureComponent {
             const events = room.getLiveTimeline().getEvents(); // timelines are most recent last
             for (let i = events.length - 1; i >= Math.max(0, events.length - maxMessagesConsidered); i--) {
                 const ev = events[i];
-                if (ev.getSender() === MatrixClientPeg.get().getUserId() || excludedUserIds.includes(ev.getSender())) {
+                if (ev.getSender() === myUserId || excludedUserIds.includes(ev.getSender())) {
```

Comment:
Somewhat redundant, actually – I see now that `excludedUserIds` is `[myUserId, SdkConfig.get()['welcomeUserId']]`. Perhaps we should explicitly check both here, or only check `excludedUserIds`?

Reply:
doh, we should only be using `excludedUserIds`.